src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -8,13 +8,10 @@
 
 import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import hudson.EnvVars;
-import hudson.FilePath;
-import hudson.Launcher;
+import hudson.*;
 import com.google.common.collect.Lists;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import hudson.Launcher.LocalLauncher;
-import hudson.Util;
```

Comment:
Don't make whitepace related changes without consultation with the maintainers.  There are enough open pull requests for the git client plugin that we can't make whitespace changes without creating conflicts in those other pull requests.

Reply:
ditto